superset-frontend/src/components/WarningIconWithTooltip/index.tsx

```diff
@@ -35,7 +35,10 @@ function WarningIconWithTooltip({
       id="warning-tooltip"
       title={<SafeMarkdown source={warningMarkdown} />}
     >
-      <Icons.AlertSolid iconColor={theme.colors.alert.base} />
+      <Icons.AlertSolid
+        iconColor={theme.colors.alert.base}
+        css={{ marginRight: `8px` }}
```

Comment:
Let's use grid units.
```suggestion
        css={{ marginRight: `${theme.gridUnit * 2}px` }}
```

Reply:
Oh, I just saw your exact suggestion - let me know if what I coded is not exactly correct. Thanks!